### PR TITLE
adding troubleshooting section

### PR DESCRIPTION
**Pull request by l0k0ms**

### CODE REVIEW

File: content/integrations/awsbilling.md [hunk 1]
```diff
@@ -30,4 +30,18 @@ AWS Budget metrics can only be collected from the AWS master account.
 
 {{< get-metrics-from-git >}}
 
-Each of the metrics retrieved from AWS will be assigned the same tags that appear in the AWS console, including but not limited to host name, security-groups, and more.
\ No newline at end of file
+Each of the metrics retrieved from AWS will be assigned the same tags that appear in the AWS console, including but not limited to host name, security-groups, and more.
+
+## Troubleshooting
+
+### No metric are reporting from my AWS billing integration
```
Comment (jeremy-lq) [inline content/integrations/awsbilling.md]: s/metric/metrics/

File: content/integrations/awsbilling.md [hunk 2]
```diff
@@ -30,4 +30,18 @@ AWS Budget metrics can only be collected from the AWS master account.
...
+Here is a check list that you can apply to troubleshoot your integration:
+
+1. Check that your IAM policy has `budgets:ViewBudget`
```
Comment (jeremy-lq) [inline content/integrations/awsbilling.md]: Ensure your IAM policy has `budgets:ViewBudget`

File: content/integrations/awsbilling.md [hunk 3]
```diff
@@ -30,4 +30,18 @@ AWS Budget metrics can only be collected from the AWS master account.
...
+2. Check if billing metrics is enabled in your payer account
```
Comment (jeremy-lq) [inline content/integrations/awsbilling.md]: Ensure billing metrics are enabled in your payer account

File: content/integrations/awsbilling.md [hunk 4]
```diff
@@ -30,4 +30,18 @@ AWS Budget metrics can only be collected from the AWS master account.
...
+3. AWS Billing metrics are collected every 4 or 8 hours by Datadog, maybe you need to wait a little bit more.
```
Comment (jeremy-lq) [inline content/integrations/awsbilling.md]: Note that AWS Billing metrics are collected every 4 or 8 hours by Datadog

File: content/integrations/awsbilling.md [hunk 5]
```diff
@@ -30,4 +30,18 @@ AWS Budget metrics can only be collected from the AWS master account.
...
+
+### `aws.billing.actual_spend` `aws.billing.forecasted_spend` `aws.billing.budget_limit` don't appear on Datadog 
+
+[Create an AWS buget](https://console.aws.amazon.com/billing/home?#/createbudget) in order to start seeing those metrics on your Datadog application. 
+Be aware that AWS Billing metrics are collected every 4 or 8 hours by Datadog.
```
Comment (jeremy-lq) [inline content/integrations/awsbilling.md]: This is point 3 above; do we need it again here (or should it be removed there)?

File: content/integrations/awsbilling.md (hunk 1 above)
Comment (l0k0ms) [reply content/integrations/awsbilling.md]: 👍

File: content/integrations/awsbilling.md (hunk 2 above)
Comment (l0k0ms) [reply content/integrations/awsbilling.md]: 👍

File: content/integrations/awsbilling.md (hunk 3 above)
Comment (l0k0ms) [reply content/integrations/awsbilling.md]: 👍

File: content/integrations/awsbilling.md (hunk 4 above)
Comment (l0k0ms) [reply content/integrations/awsbilling.md]: 👍

File: content/integrations/awsbilling.md (hunk 5 above)
Comment (jeremy-lq) [reply content/integrations/awsbilling.md]: Sounds reasonable. LGTM

**Pull request merged**